setup.py

```diff
@@ -67,9 +67,11 @@
     url="https://github.com/ApeWorX/silverback",
     include_package_data=True,
     install_requires=[
-        "eth-ape>=0.6.19,<1.0",
-        "taskiq[metrics]>=0.6.0,<0.7.0",
         "click",  # Use same version as eth-ape
+        "eth-ape>=0.7.0,<1.0",
```

Comment:
question: we're breaking compat with 0.6?

Reply:
Actually, I think this is necessary.  We have to because of how taskiq decides which pydantic version to use.